### PR TITLE
developing

### PR DESCRIPTION
**Pull request by Naroru**
I added new features

### CODE REVIEW

**Comment by nerevar51** [comment]
> I added new features

Hello

File: src/main/java/Main.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-//ыщьу сщьь
+//developing
```
Comment (nerevar51) [inline src/main/java/Main.java]: You should change it to Developing 2

File: src/main/resources/someFile.txt [hunk 2]
```diff
@@ -1 +1 @@
-Hello2
\ No newline at end of file
+developing
```
Comment (nerevar51) [inline src/main/resources/someFile.txt]: here change to developing 3

**Review by nerevar51 — CHANGES_REQUESTED** [review]

File: src/main/java/Main.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-//ыщьу сщьь
+//developing 2
```
Comment (Naroru) [inline src/main/java/Main.java]:
```suggestion
//developing 3
```
sorry, it should be developing 3 also

**Review by nerevar51 — APPROVED** [review]

**Pull request merged**